src/app/globals.css

```
@@ -127,75 +129,358 @@ html.dark ::selection {
+    background-position: 200% 0;
+  }
 }
 
 /* Gentle pulsing core glow */
 @keyframes fate-core {
-  0%, 100% { box-shadow: 0 0 0px 0 currentColor; opacity: .25; }
-  50%      { box-shadow: 0 0 40px 6px currentColor; opacity: .35; }
+
+  0%,
+  100% {
+    box-shadow: 0 0 0px 0 currentColor;
```

Stylelint (17.4.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)